app/assets/javascripts/app/review_form.js

```diff
@@ -93,7 +93,11 @@ App.addChild('ReviewForm', _.extend({
   },
 
   validateZipAndPhone: function() {
-    return (this.validatePhone(this.$phone) && this.validateMasked(this.$zip));
+    if(this.$country.val() == 'BR'){
+      return (this.validatePhone(this.$phone) && this.validateMasked(this.$zip));
+    }else{
+      return this.validateMasked(this.$zip)
```

Comment:
não precisa desse `this.validateMasked(this.$zip)` no `else`, quando o país n é BR as máscaras tanto do telephone como do zip são removidas e os campos deixam de ser obrigatórios